phoenixRest/models/core/event.py

```diff
@@ -69,32 +68,37 @@ class Event(Base):
 
     cancellation_reason = Column(Text)
 
-    def __init__(self, name: str, start_time: DateTime, end_time: DateTime, max_participants: int):
+    def __init__(self, name: str, start_time: DateTime, end_time: DateTime, booking_time: DateTime, priority_seating_time_delta: int, seating_time_delta: int, max_participants: int, participant_age_limit_inclusive: int, crew_age_limit_inclusive: int, theme: str, location_uuid: str, seatmap_uuid: str):
         self.name = name
         self.start_time = start_time
         self.end_time = end_time
-        self.booking_time = start_time - timedelta(days=31)
-        self.priority_seating_time_delta = 60*30
-        self.seating_time_delta = 60*60
+        self.booking_time = booking_time
+        self.priority_seating_time_delta = priority_seating_time_delta
+        self.seating_time_delta = seating_time_delta
         self.max_participants = max_participants
+        self.participant_age_limit_inclusive = participant_age_limit_inclusive
+        self.crew_age_limit_inclusive = crew_age_limit_inclusive
+        self.theme = theme
+        self.location_uuid = location_uuid
+        self.seatmap_uuid = seatmap_uuid
 
 
     def __json__(self, request):
         return {
             'name': str(self.name),
             'uuid': str(self.uuid),
-            'participant_age_limit_inclusive': self.participant_age_limit_inclusive,
-            'crew_age_limit_inclusive': self.crew_age_limit_inclusive,
+            'start_time': int(self.start_time.timestamp()),
+            'end_time': int(self.end_time.timestamp()),
             'booking_time': int(self.booking_time.timestamp()),
             'priority_seating_time_delta': self.priority_seating_time_delta,
             'seating_time_delta': self.seating_time_delta,
-            'start_time': int(self.start_time.timestamp()),
-            'end_time': int(self.end_time.timestamp()),
-            'theme': self.theme,
             'max_participants': self.max_participants,
-            'cancellation_reason': self.cancellation_reason,
+            'participant_age_limit_inclusive': self.participant_age_limit_inclusive,
+            'crew_age_limit_inclusive': self.crew_age_limit_inclusive,
+            'theme': self.theme,
+            'location_uuid': self.location_uuid,
             'seatmap_uuid': self.seatmap_uuid,
-            'location': self.location
+            'cancellation_reason': self.cancellation_reason
```

Comment:
This re-organizing makes it harder to read the diff, why was things re-organized?